src/modules/TaskGenerate.js

```diff
@@ -0,0 +1,50 @@
+import tasks from './tasks.js';
+// import trashIconPng from '../assets/trash-icon.png';
+import * as vars from './variables.js';
+
+export default class TaskGenerate {
+  constructor(text) {
+    this.text = text;
+    this.completed = false;
+    this.id = tasks.length + 1;
+  }
+
+  static addTask(text) {
+    const task = new TaskGenerate(text);
+    tasks.push(task);
+    this.reloadTasks();
+  }
+
+  static reloadTasks() {
+    const tasksContainer = document.querySelector('tasks-container');
+    console.log(tasksContainer);
```

Comment:
it will be better to remove console log from your code because doesn't show best practice